### PR TITLE
Peer Review Pull Request

### PR DESCRIPTION
**Pull request by mzmarlies**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,59 +1,30 @@
...
-import axios from 'axios';
-
-// Components:
-import TileOne from './components/TileOne';
-// import TileTwo from './components/TileTwo';
-
+// STYLES:
+import './styles/sass/App.scss';
+// COMPONENTS:
+import ApiImages from './components/ApiImages';
```
Comment (harrisonhui) [inline src/App.js:4]: Loved how your components flowed from the parent App.js to ApiImages to originalTiles to SingleTile

File: src/components/SingleTile.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    let matchedCriteria 
+    if(props.onePhoto.match === true){
+        matchedCriteria = true
+    } else if(flipState === true){
+        matchedCriteria = true
+    } else {
+        matchedCriteria = false
+    }
+
+    console.log(props)
```
Comment (harrisonhui) [inline src/components/SingleTile.js:30]: I notice I do this a lot as well, but maybe we can delete any outstanding console.log or comments on the code in the final push to production branch and keep the other variant with console.log + comments in another branch?

File: src/styles/sass/App.css [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  text-decoration: none;
+  color: #ff8add;
+  text-shadow: 0.1px 0.1px white;
+}
+
+.App {
+  text-align: center;
+}
+
+.App-logo {
```
Comment (harrisonhui) [inline src/styles/sass/App.css:121]: I think these are the default React styling (could be completely wrong) but if it is, could also delete these base styling from the CSS file.

**Review by harrisonhui — CHANGES_REQUESTED** [review]
Code was awesome, the flow of your components was great, and loved how you were able to incorporate sass in a react project as well!

Most of my comments revolved around deleting comments or console.log to remove some clutter, but apart from that amazing job!

File: src/components/SingleTile.js (hunk 2 above)
Comment (mzmarlies) [reply src/components/SingleTile.js:30]: good idea!:)

**Pull request closed without merging**